models/friend/index.js

```diff
@@ -216,18 +231,30 @@ async function blockUser(uId, notUId) {
 async function unblockUser(uId, notUId) {
   try {
     // needs to be changed to reflect situations where ids are swapped (see delete friend request query)
-    var queryString = 'UPDATE `request` SET `blocked`=0 ' +
+    var queryString = 'DELETE FROM `request` ' +
       'WHERE (`receiver_id`=? AND `sender_id`=?) ' +
       'OR (`receiver_id`=? AND `sender_id`=?)';
     const results = await DB.query(queryString,
       [uId, notUId, notUId, uId ]);
-    return results;
+    const auxResults = await unblockUserAux(uId, notUId);
+    return auxResults;
   }  catch (error) {
     Logger.error(error);
     throw error;
   }
 }
 
+async function unblockUserAux(uId, notUId) {
+  try {
+    // needs to be changed to reflect situations where ids are swapped (see delete friend request query)
+    var queryString = 'DELETE FROM `blocked` WHERE `blocker_id`=? AND `blockee_id`=? ';
```

Comment:
Since this is one-line, can we try 
```
const queryString = `DELETE FROM \`blocked\` WHERE \`blocker_id\`=${uId} AND \`blockee_id\`=${notUId} ;`;
const results = await DB.query(queryString);
```